### ISSUE TITLE
Bug on Responsible Party people type - can't edit attachments (TL) (With CTM but waiting on examples from ROW)

### ISSUE THREAD

**Issue by TracyLinder** [cityofaustin/atd-data-tech#14097]
### ROW reporting that users cannot edit existing attachments. 
- INC0331474 created

**Comment by TracyLinder** [comment]
@KyleErich FYI - i created a new ticket for this.

**Comment by Nadin-Nader** [comment]
**INC0331474 created**

**Comment by TracyLinder** [comment]
Victor asked me for examples; I chatted Kyle asking for it.

**Comment by TracyLinder** [comment]
### Outcome & Closing

- Jenice added this functionality to the Streets & Spaces security group. She said it would be too much work to create a new group for this function or to add it to all existing groups, so she put it in Streets & Spaces for now. FYI @Nadin-Nader 

Kyle tested successfully.
